docs/blog/2019-08-06-gazette-august/index.md

```diff
@@ -0,0 +1,124 @@
+- [grantglidewell](https://github.com/grantglidewell) added [support](https://github.com/gatsbyjs/gatsby/pull/14630) for Gatsby [Preview](https://www.gatsbyjs.com/preview) for Drupal in `gatsby-source-drupal`
+- [elken](https://github.com/elken) added [support for the Prism command-line](https://github.com/gatsbyjs/gatsby/pull/16170) plugin in `gatsby-remark-prismjs`
+- [cephalization](https://github.com/cephalization) added a [great error check](https://github.com/gatsbyjs/gatsby/pull/16272) in Gatsby
+
+Thank you for all your contributions, folks!
+
+## Gatsby Site and Theme of the month
+
+Time to show off some great stuff people have been building
+
+### Site of the Month
+
+The Gatsby site of the month is [www.fourpost.com](https://www.fourpost.com)
+
+We love it because:
+
+- It's so fast
+- The illustrations are so very pretty
+- Uses Gatsby Preview
+- Pulls in content from DatoCMS & EventBrite
+
+### Theme of the Month
+
+The Gatsby theme of the month is [Novela (a theme by Narative)](https://github.com/narative/gatsby-theme-novela)
+
+![Novela (a Gatsby theme by Narative)](./gatsby-theme-novela-hero.jpg)
+
+With minimal styling and maximum features — including multiple homepage layouts, built-in social sharing and dark mode — Novela makes it easy to start publishing beautiful articles and stories with Gatsby.
+
+I'm a big fan of the team at Narative and think Novela shows off some of the best part of Gatsby themes.
+
+Here is what [Dennis Brotzky](https://twitter.com/_brotzky) (Co-founder, [narative.co](https://www.narative.co/)) had to say:
+
+> We’ve always enjoyed using Gatsby for our projects and when themes were announcement we knew it would be special. Novela is our first dive into themes and we’ve quickly grown to love it. It’s exciting to be part of the formation of a new way to build and share Gatsby sites.
+
+## Community Maintainers Meeting
+
+The Core Maintainers Meeting is now the **Community** Maintainers Meeting.
+
+We've renamed the meeting to emphasise that this time is for the community. You can join the meeting whether you've contributed to Gatsby, use it regularly or just tried it out for the first time. Everyone is welcome!
+
+We've also restructured the meeting with a new format that includes
+
+- time for questions
+- time for updates
+- live pull request reviews
+
+We think live pull request reviews are a great way to involve the community in not only contributing code but also helping us merge those contributions in. We hope these give you some insight into how we review contributions and help make [your first contribution](http://gatsby.dev/pair-programming) even easier.
+
+In case you're not comfortable asking questions on the call (first call with strangers can be intimidating, we understand), we have a form to submit questions/pull requests for live review. The form is at https://gatsby.dev/oss-maintainers-form.
```

Comment:
Removed _first_, I've been in loads of these and haven't lost the fear :)

```suggestion
In case you're not comfortable asking questions on the call (a video call with strangers can be intimidating, we understand), we have a form to submit questions/pull requests for live review. The form is at https://gatsby.dev/oss-maintainers-form.
```

Reply:
Haha true!